### PR TITLE
propositions

### PR DESCRIPTION
**Pull request by AymericPlanche**
Re-organize doc.
Remove duplicates.
Add some TODOs

### CODE REVIEW

File: content/akeneo-event-platform/akeneo-event-platform.md [hunk 1]
```diff
@@ -2,36 +2,11 @@
...
 ## Akeneo Event Platform Overview
 
-The **Akeneo Event Platform (AEP)** seamlessly integrates with Akeneo PIM through a subscription-based event system. External applications can subscribe to specific events triggered within Akeneo PIM, enabling real-time data updates and synchronization.
+The **Akeneo Event Platform (AEP)** seamlessly integrates with Akeneo PIM through a subscription-based event system. External applications can subscribe to specific events triggered within Akeneo PIM, enabling faster data updates and synchronization than continuous polling.
 
 AEP ensures smooth and efficient responses to changes in your PIM, such as product updates, deletions, and other significant events, enhancing your system’s responsiveness and connectivity.
 
-## What AEP Is Not
-
-To set the right expectations, here's what AEP is **not**:
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/akeneo-event-platform.md:11]: transformed as FAQ.

Having such "negative" statements in the first page is pretty confusing

File: content/akeneo-event-platform/akeneo-event-platform.md [hunk 2]
```diff
@@ -2,36 +2,11 @@
...
-- Access to the PIM for the authentication credential
-- For subscription destinations
-  - **`PubSub`** & **`HTTPS`** subscription are allowed. ~~HTTP~~ **is not supported**.
-  - For PubSub destination, make sure you meet [the requirements](/akeneo-event-platform/concepts.html#pub-sub-subscription)
-  - For HTTPS destination, make sure you meet [the requirements](/akeneo-event-platform/concepts.html#https-subscription)
-
-- You can have up to 20 subscribers for each PIM instance.
-- Each subscriber can have up to 20 subscriptions.
-
-### Authentication
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/akeneo-event-platform.md:31]: getting started description and platform requirements is sufficient

File: content/akeneo-event-platform/akeneo-event-platform.md [hunk 3]
```diff
@@ -2,36 +2,11 @@
...
-## What AEP Is Not
-
-To set the right expectations, here's what AEP is **not**:
-
-- **Not a Data Storage Solution**: AEP streams real-time events but does not store event data permanently.
-- **Not a Replacement for Akeneo PIM API**: AEP complements the Akeneo PIM API by offering event-driven capabilities but does not replace its core functionalities.
-- **Not a Business Logic Handler**: AEP transmits events to subscribed endpoints but does not handle complex business logic or workflows.
-- **Not the v2 of the Akeneo Event API**: AEP is a new platform with distinct functionalities and should not be considered as the next version of [Akeneo's Event API](https://api.akeneo.com/events-documentation/overview.html). AEP offers more granular event handling compared to the Event API.
-
-## Usage Conditions
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/akeneo-event-platform.md:18]: duplicate

File: content/akeneo-event-platform/best-practices.md [hunk 4]
```diff
@@ -1,29 +1,27 @@
 # Best practices
 
-## Queuing Events
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/best-practices.md:3]: already explained in "concepts -> key behavior"

File: content/akeneo-event-platform/concepts.md [hunk 5]
```diff
@@ -160,44 +162,46 @@ Example of an event payload for a productDeleted event
...
 
-## Retry policies
-
-Ensure reliable message processing in distributed systems. Retry policies help manage transient failures and guarantee message delivery even when issues occur.
+## Key platform behaviors
 
-Here, we outline the key **retry policies** implemented in our system.
+Your subscribing service implementation and architecture must deal with the following capabilities and constraints to consume events from the platform at its best.
 
-### ACK timeout
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/concepts.md:169]: "ack" terminology is pretty "pubsub" tainted.

changed to the notion of "delivery timeout"

File: content/akeneo-event-platform/faq.md [hunk 6]
```diff
@@ -1,47 +1,60 @@
 # FAQ
-### What are the limits of subscriptions & subscribers?
+### Is their any quotas or limits ?
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/faq.md:2]: content to be adapted in coming days

File: content/akeneo-event-platform/faq.md [hunk 7]
```diff
@@ -1,47 +1,60 @@
...
+- HTTPS - generic, can be considered as a Webhook feature
+- Google Cloud Pub Sub
 
-Working under the assumption of at least once delivery, especially within the context of an event-driven architecture, is a responsible approach to reliably processing events, and this is what we want to provide to our customers and partners. 
-
-**Why "At-Least Once" Delivery?**
-An event may be sent multiple times in the "at least once" delivery model. This can happen if the event platform doesn't receive a confirmation of successful receipt from the destination (such as an HTTP 200 status or a Pub/Sub acknowledgement). For instance, we send a message that is received and processed by the destination, but if the event platform does not get the acknowledgement, it will retry sending the message to ensure delivery.
+We will add other subscription channels based on feedback. Please [fill-in this form](https://forms.gle/XsZ7rovRnqfAn4xF9) to propose & upvote new destination types.
```
Comment (AymericPlanche) [inline content/akeneo-event-platform/faq.md:25]: added a google form link to gather feedback easily, wdyt ?

**Comment by akeneo-secops** [comment]
### Gitleaks PR check
🛑 **We detected secrets within this PR**<br/>If this detection is a **true positive**, please rotate the secret **ASAP**.<br/><br/>If this secret is a false positive, you can add the corresponding fingerprint<br/>to your <code>.gitleaksignore</code> file and commit the change to this branch. More details in [Notion](https://www.notion.so/akeneo/Prevent-secrets-leakage-cad5e759ccbd436cb4a7966db3203e94).<br/><br/><details>- [Fingerprint f84f63cf768427aa8991c9b618a38fe66fa6bf50:content/akeneo-event-platform/getting-started.md:hashicorp-tf-password:37](https://github.com/akeneo/pim-api-docs/blob/f84f63cf768427aa8991c9b618a38fe66fa6bf50/content/akeneo-event-platform/getting-started.md#L37)</br></details>

**Review by AdelELHAIBA-Akeneo — APPROVED** [review]

**Review by guillaumefer-akeneo — APPROVED** [review]

File: content/akeneo-event-platform/faq.md [hunk 8]
```diff
@@ -1,47 +1,69 @@
...
+Akeneo Event Platform is a new platform with distinct functionalities and should not be considered as the next version of [Akeneo's Event API](https://api.akeneo.com/events-documentation/overview.html). 
+
+The new platform offers more granular event handling compared to the Event API, every events will be tried to be delivered and retried in case of failure.
+
+### My subscription is repeatedly suspended, I resume it all the time and I loose events, why ?
+
+According to our delivery timeout strict policy, we want your subscribing service to acknowledge the reception of the event as fast as it can in order for our platform to deliver events continuously.
+
+Resuming suspended subscription several times indicates than you might change the way you deals with event consumption.
+
```
Comment (AdelELHAIBA-Akeneo) [inline content/akeneo-event-platform/faq.md:68]: @AymericPlanche j'ai pensé au filtrage IP qu'on pourrait mettre dans la FAQ aussi

Comment (AdelELHAIBA-Akeneo) [reply content/akeneo-event-platform/faq.md:68]: Ah j'ai vu en daily que tu as créé le ticket :heart_eyes:

**Pull request merged**